client/src/components/molecules/NotificationList/index.tsx

```diff
@@ -120,7 +106,9 @@ const NotificationList: FC<Props> = (props): JSX.Element => {
                         )}
                       </span>
                       <span className="font-semibold text-slate-900 underline line-clamp-1">
-                        {notification.data.task?.name}
+                        {notification.data.type === 'assignTask' && notification.data.task?.name}
+                        {notification.data.type === 'commit' && notification.data.repository?.name}
+                        {notification.data.type === 'merge' && notification.data.pr_details?.title}
```

Comment:
ADD constants instead of hard coding `assignTask`, `commit` and `merge`.
Create Separate file for Constants.
const NotificationTypes {
 ASSIGN_TASK: 'assignTask`
 COMMIT: 'commit'
 MERGE: 'merge'
}